esmvaltool/diag_scripts/ocean/diagnostic_timeseries.py

```diff
@@ -184,9 +241,16 @@ def multi_model_time_series(
             else:
                 color = 'blue'
 
+            # Take a moving average, if needed.
+            if 'moving_average' in cfg.keys():
```

Comment:
`if member in cfg` is enough, not a good practice to iterate on keys or values, `dict.items()` is used if you want to iterate over `key, value`

Reply:
No sure I understand what you mean, are you suggesting that we change:
``` 
if 'moving_average' in cfg.keys():
```
to:
```
if 'moving_average' in cfg:
```
?

Reply:
yup